### ISSUE TITLE
deb doesn't work for Ubuntu 22.10

### ISSUE THREAD

**Issue by kokizzu** [lyrebird-voice-changer/lyrebird#113]
![image](https://user-images.githubusercontent.com/1061610/235198403-36d5c16b-323f-4b74-b5d2-40f67d3a2a22.png)
```
sudo dpkg -i lyrebird_1.1.0-3.deb  

dpkg: dependency problems prevent configuration of lyrebird:
 lyrebird depends on sox; however:
  Package sox is not installed.
 lyrebird depends on pulseaudio; however:
  Package pulseaudio is not installed.
 lyrebird depends on libsox-fmt-pulse; however:
  Package libsox-fmt-pulse is not installed.

sudo apt install sox pulseaudio libsox-fmt-pulse             

You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 pipewire-alsa : Conflicts: pulseaudio
                 Conflicts: pulseaudio:i386
 pop-desktop : Conflicts: pulseaudio
               Conflicts: pulseaudio:i386
```

**Comment by atraxsrc** [comment]
I use pop too, download the tar file and extract. CD into lyrebird and run > python3 app.py

**Comment by xsabatox** [comment]
I can confirm that the .deb file installation error persists. 

@atraxsrc, using your solution I had to pip install toml and apt install sox manually. Then, it throws warnings about no PulseAudio daemon running, or not running as session daemon.

**Comment by harrego** [comment]
This has now been [fixed in v1.2.0](https://github.com/lyrebird-voice-changer/lyrebird/releases/tag/v1.2.0). Please open a new issue if you continue to run into issues!
